static/css/main.css

```diff
@@ -1,77 +1,88 @@
 .right {
-    float: right;
+    float  : right;
```

Comment:
eww this CSS code style is gross

Reply:
I blame my auto-formatter

Reply:
Okay I've found a better formatter [here](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)